proto/substrait/algebra.proto

```diff
@@ -648,11 +647,27 @@ message Expression {
     //  - Required enum arguments must be bound using FunctionArgument.enum
     //    followed by Enum.specified, with a string that case-insensitively
     //    matches one of the allowed options.
-    //  - Optional enum arguments must be bound using FunctionArgument.enum
-    //    followed by either Enum.specified or Enum.unspecified. If specified,
-    //    the string must case-insensitively match one of the allowed options.
     repeated FunctionArgument arguments = 4;
 
+    // Options to specify behavior for corner cases, or leave behavior
+    // unspecified if the consumer does not need specific behavior in these
+    // cases.
+    repeated FunctionOption options = 5;
+    message FunctionOption {
```

Comment:
Why declare this in each function type? FunctionArgument is external and it seems like this should similarly be so.

Reply:
Good point.  I've moved this so it is declared once, next to `FunctionArgument`.